### PR TITLE
Removed deprecated labels for 1.19+ clusters

### PR DESCRIPTION
**Pull request by aclevername**
### Description
Since `1.14` the `beta.kubernetes.io/arch` and `beta.kubernetes.io/os` labels have been deprecated, in 1.19 they started to replace it with `kubernetes.io/arch` and `kubernetes.io/os`. This PR updates it so that the correct labels are used for resources specific to 1.19, and for resources that cannot be updated by eksctl both the old and new labels are provided.

https://v1-19.docs.kubernetes.io/docs/setup/release/notes/

https://github.com/weaveworks/eksctl/issues/3562

### Checklist
- [ ] Added tests that cover your change (if possible)
- [ ] Added/modified documentation as required (such as the `README.md`, or the `userdocs` directory)
- [ ] Manually tested
- [ ] Made sure the title of the PR is a good description that can go into the release notes
- [ ] (Core team) Added labels for change area (e.g. `area/nodegroup`) and kind (e.g. `kind/improvement`)

### BONUS POINTS checklist: complete for good vibes and maybe prizes?! :exploding_head:
- [ ] Backfilled missing tests for code in same general area :tada:
- [ ] Refactored something and made the world a better place :star2:

### CODE REVIEW

File: integration/data/windows-server-iis.yaml [hunk 1]
```diff
@@ -28,5 +28,5 @@ spec:
         - -command
         - "Add-WindowsFeature Web-Server; Invoke-WebRequest -UseBasicParsing -Uri 'https://dotnetbinaries.blob.core.windows.net/servicemonitor/2.0.1.6/ServiceMonitor.exe' -OutFile 'C:\\ServiceMonitor.exe'; echo '<html><body><br/><br/><marquee><H1>Hello EKS!!!<H1><marquee></body><html>' > C:\\inetpub\\wwwroot\\default.html; C:\\ServiceMonitor.exe 'w3svc'; "
       nodeSelector:
-        beta.kubernetes.io/os: windows
+        kubernetes.io/os: windows
```
Comment (aclevername) [inline integration/data/windows-server-iis.yaml:31]: integration tests use 1.19 now.

File: pkg/addons/assets/vpc-admission-webhook-dep.yaml [hunk 2]
```diff
@@ -33,9 +33,28 @@ spec:
...
+                - amd64
+            - matchExpressions:
+              - key: kubernetes.io/os
+                operator: In
+                values:
+                - linux
+              - key: kubernetes.io/arch
+                operator: In
+                values:
+                - amd64
```
Comment (aclevername) [inline pkg/addons/assets/vpc-admission-webhook-dep.yaml:57]: unlike coredns,kubeproxy and aws node we don't offer the user to update the deployment. In this case I think having both labels supplied makes sense, it prevents any users experiencing breaking changes during an upgrade.

**Comment by aclevername** [comment]
integration test run: https://github.com/weaveworks/eksctl/actions/runs/744459905

**Review by Callisto13 — APPROVED** [review]
lgtm

**Pull request merged**